### PR TITLE
Add recipe for readuct

### PR DESCRIPTION
**Pull request by awvwgk**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by awvwgk** [comment]
@conda-forge/help-c-cpp This recipe is ready for review.

**Comment by awvwgk** [comment]
@conda-forge/staged-recipes This recipe is ready for review

File: recipes/scine-readuct/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  name: {{ pkg_name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/qcscine/readuct/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 55ab11c432ff0bae9521c561256db2fe90dfb0aad31d934eefe3874e42749ba3
+    fn: {{ name }}.tar.gz
+  - url: https://github.com/qcscine/development-utils/archive/refs/tags/4.0.0.tar.gz
+    sha256: 54002c2082b6bb75672ec66bf9cf3935bbdf6b085ed9b4d7174cbdedb7c2275d
+    fn: dev.tar.gz
```
Comment (wolfv) [inline recipes/scine-readuct/meta.yaml:15]: this should not be needed actually

File: recipes/scine-readuct/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ pkg_name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/qcscine/readuct/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 55ab11c432ff0bae9521c561256db2fe90dfb0aad31d934eefe3874e42749ba3
+    fn: {{ name }}.tar.gz
```
Comment (wolfv) [inline recipes/scine-readuct/meta.yaml:12]: this shoudl not be needed.

**Comment by wolfv** [comment]
Looks good to me

**Comment by awvwgk** [comment]
@wolfv Requested changes have been implemented.

**Review by carterbox — CHANGES_REQUESTED** [review]
[Link checker reports that](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=498949&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=3747) `libblas` and `openmp` are not being linked against on the OSX build. Please switch to dynamic linking or remove these dependencies for that platform.

**Pull request merged**